scripts/release-merge-to-main.ts

```diff
@@ -77,11 +80,15 @@ async function ensureBranch(owner: string, repo: string, branchName: string, tag
       await git.updateRef({
         owner, repo, ref, sha,
       });
+    } else {
+      console.log(`Branch ${ owner }/${ repo }/${ ref } is already up-to-date.`);
     }
   } catch (ex) {
     if (!(ex instanceof RequestError) || ex.status !== 404) {
       throw ex;
     }
+    console.log(`Creating new branch ${ owner }/${ repo }/${ ref }` +
```

Comment:
Hmm, this didn't pick up 8f782de061fa490ffe1eca2bafd3a8414933e6b6 — I hope merge won't have issues.

Reply:
Sorry, my mistake. I've cherry-picked it as well now.